### ISSUE TITLE
Cannot compile 3.1.0 with --enable-hts

### ISSUE THREAD

**Issue by birnbera** [smithlabcode/preseq#48]
It looks like there were some recent header file name changes that may be causing this. The compiler throws the error:
```
./src/load_data_for_complexity.cpp:184:10: fatal error: htslib_wrapper.hpp: No such file or directory
 #include <htslib_wrapper.hpp>
```
Indeed, it seems `htslib_wrapper.hpp` (and `.cpp`) is missing, although the file `htslib_wrapper_deprecated.hpp` is present. What's strange is that the `preseq` repo contains a submodule pointing to a commit that does have both files present, so I'm not sure why it's missing or what else might be going on here.

Any suggestions would be much appreciated!

**Comment by andrewdavidsmith** [comment]
@birnbera Looks like we do have a problem. Should be able to fix it pretty quick. Expect an updated release very soon.

**Comment by birnbera** [comment]
Thanks! FYI, I was able to build and use the tools just be deleting the offending include line in `load_data_for_complexity.cpp`.

**Comment by andrewdavidsmith** [comment]
Well, that's pretty much the defining change in version 3.1.1 (coming very soon)...

**Comment by terencewtli** [comment]
Sorry for the error! We have just released 3.1.1 which should fix this issue.
